app/views/users/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class = "user-profile">
+  <h1> <%= @user.username %> </h1>
+  <div>
+    <%=  image_tag "pajama_man.jpg"%>
+  </div>
+  <div>
+    <ul>
+      <li> Location:<%= @user.location %> </li>
+      <li> Age: <%= @user.age %> </li>
+      <li> <%= @user.registered_at %> </li>
+    </ul>
+  </div>
+  <%= @user.bio %>
+</div>
+
+<div class = "user-questions">
+  <%= @user.questions.count%>
+  <ul>
+  <% @user.questions.each do |question|%>
+    <li><%= question.title %>
+    <%= question.votes.count %></li>
```

Comment:
Suggestion: investigate a `counter_cache` column. Search for `counter_cache` here: http://guides.rubyonrails.org/association_basics.html#belongs-to-association-reference